servicedef/sdk_config.go

```diff
@@ -26,7 +26,7 @@ type SDKConfigParams struct {
 	ClientSide          o.Maybe[SDKConfigClientSideParams]          `json:"clientSide,omitempty"`
 	Hooks               o.Maybe[SDKConfigHooksParams]               `json:"hooks,omitempty"`
 	Wrapper             o.Maybe[SDKConfigWrapper]                   `json:"wrapper,omitempty"`
-	DataSystem          o.Maybe[SDKConfigDataSystem]                `json:"dataSystem,omitempty"`
+	Persistence         o.Maybe[SDKConfigPersistence]               `json:"persistence,omitempty"`
```

Comment:
I just noticed the `PersistentDataStore` config in this struct. Hmm. We could just use that I guess? it seems to not be used.

Reply:
I will de-dupe as needed.